### ISSUE TITLE
The CI for unit test is failing

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-web-map-core#149]
https://github.com/Greenstand/treetracker-web-map-core/actions/runs/3627876711/jobs/6118238700

Seems some syntax proble? can not find `cy` by cypress

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-core#treetracker-web-map-core) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.

**Comment by dadiorchen** [comment]
@keyy123 do you now why this happens?

**Comment by keyy123** [comment]
I can look into it! @dadiorchen

**Comment by dadiorchen** [comment]
@keyy123 thank you!

**Comment by keyy123** [comment]
I found the issue and will make a PR today. Issue: Tests files including the ones in cypress have `.spec.js` file extensions which jest detects automatically. The solution is to change naming conventions for cypress test files in web-map-core similar to how it is done in web-map-client.

TLDR: .spec.js => cy.js
